UnitTests/PowerShell/LISAv2-Framework.Tests.ps1

```diff
@@ -87,8 +87,8 @@ Describe "Test if ${moduleName} Run-LISAv2 fails to parse report results on Azur
 		Mock New-ZipFile -Verifiable -ModuleName $moduleName { return }
 
 		{ Run-LISAv2 -Verbose -TestPlatform "Azure" -RGIdentifier "test" -ARMImageName "one two three four" `
-			-TestLocation "westus2" -TestNames "ETHTOOL-CHANGE-RINGBUFFER"} | Should Throw
+			-TestLocation "westus2" -TestNames "VERIFY-DEPLOYMENT-PROVISION"} | Should Throw
```

Comment:
this is a generic/standard test that should exist, default to a more common one by name. Only for PR checker, does not affect anything else